pkg/controller/resourcegroup/controller_cleanup.go

```diff
@@ -40,8 +40,12 @@ func (r *ResourceGroupReconciler) cleanupResourceGroup(ctx context.Context, rg *
 		return fmt.Errorf("failed to shutdown microcontroller: %w", err)
 	}
 
+	group := rg.Spec.Schema.Group
```

Comment:
I understand what's being done here but I'm just curious why the `group` isn't populated with `kro.run` by default because in the schema we've specified this: https://github.com/kro-run/kro/blob/main/api/v1alpha1/types.go#L67

```
// +kubebuilder:default="kro.run"
```

cc @a-hilaly

Reply:
But I already added the markers in my PR 🤔 that's why I'm confused why this was needed, were the markers not working this means? 

You can see they're already in the code base: https://github.com/kro-run/kro/blob/main/api/v1alpha1/types.go#L67